### PR TITLE
Odie Client: add Zendesk conversation to chat

### PR DESCRIPTION
**Pull request by arcangelini**
## Proposed Changes

* Add Smooch hook to Odie
* Add support provider state to manage chat between Odie and Zendesk
* Add feature flag to hide behind

## Why are these changes being made?

We are unifying the support chat experience. This involves merging the AI conversation (odie) and the human conversation (zendesk) into a singular chat window (odie-client). This is a very raw version of the chat experience. There are some known bugs and upgrades that need to be made. We are deploying in this state to allow others to work asynchronously.

## Testing Instructions

The main thing to test is for regression. All of the changes should exist behind the feature flag, but we need to make sure that the experience without the flag is the exact same as production. We should also be sure there are no additional requests to Zendesk.

To test the new chat experience add `?flags=help-center-experience` to the URL. It would be easiest using the Live Link probably.
Then start a conversation with odie and ask to talk to a human.
Click the Get Support button and you should get a new conversation within the Help Center.

### CODE REVIEW

**Comment by arcangelini** [comment]
Issues to address:

- [ ] Make sure we are talking to the right ticket. I have seen inconsistencies in retrieving the wrong conversation and replying to a closed ticket.
- [ ] Make sure we are listening to the right conversation. If the user gets a reply from an HE on ticket B while the chat is open to ticket A the response is picked up.

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/16912202

Some locales (Hebrew, Japanese) have been temporarily machine-translated due to translator availability. All other translations are usually ready within a few days. Untranslated and machine-translated strings will be sent for translation next Monday and are expected to be completed by the following Friday.

Hi @arcangelini, could you please edit the description of this PR and add a screenshot for our translators? Ideally it'd include all of the following strings:
- `AI profile picture`
- `Type a message…`

Thank you in advance!

**Comment by manzoorwanijk** [comment]
This PR seems to have broken the local `yarn install` - p1729157088800809-slack-C02DQP0FP

File: packages/odie-client/src/assets/human-avatar.tsx [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
...Gp2CYszJPMki9iZ0vzY40nHlmGR8pErcrwezzy1mpxLm1+gebnSv3zMXW3KStWva/qgUlqRzLfd8oaci2Yu2YULHF9MK3NOGO7ut81CixmGQRfbwNXx++JsnKmsIu9nsIHqVOYlmDczT7p0te/eRjYSH2pSkIxaIq3UzwkLWe141EKq4NGHNvU1LxdOCLq4BtnMfp50NAE5Q484NFDBVVSf5ZjljQccBWUM7FDHDW2y6dBdTBVXxdaPjurYAilr0nJ9slUHSvmzEKKx7Cl5WVN/YFfcY7n+ZBC9ibIGYTL7wfJked6Z4Rp7SwxMA/97gCiURXfQ3TKBu2ump0TvAXmToI6KUV57mksxJXov2EzgdosZhrt/xveU6D0iN6vzDty9zDCcEr0XDI4gi8p5uOwqgTsleu/IFUhGkCdw7a6MvCnR+4IgDhNhwrKFfyrRw8dmsa8jCAm6iN25KHJK9H4hQNHaraIyKrA911OiDwTJ0CEm7+5+qKaeEn0gCKgU3dn5wQJqwh8xNelQiMP0tnYf36dAPN6hnBvnNLijjrwCQqUEXXT3wXPV8mNMJcA7bB4zm8yg2SRDihhvvivaGm5gVkIu38agBbW8SKtZlLJNsR/kXCeYENND2dJRlILSmRpLc6gPPsrSAs5jj+6w1zFi06YGFROUu2KuQJzDOpZm+OAy6pMf4NwpRArtMcV+UCZwdQe9gfSLitac0vPHePoM6qlHOLdEpQrTHfEgyHNdWx9C5yAwnJ/l40uoMws8cZH3XcQY+vG4l3tEMaBATL943nCUEhieWOKpJS4ewzQqfPxxVu6ycpvL16kGqKkTsz9sCnWbtEHW4MIyf/N+nj5Dy2CAhWN87kNcu0U/4a1VjKFW3eEzp3gwBCxJh1hx7hJfvsRfXmCpDptZ8LOL/PVnUMJPfsGN2/RjELRCa2RqX+8Ij/PY3B1xVKqcPc7nH+Mrj3NupjikINpoPnCO403OLvEfP+X1KyRZOTI0V0BTrreDx3vEo0ApKnXed4FnPsXnnuLMApXyKPFbHZXVDV54jd9c4eoqV29x+w5hj8wiw25Cf5cgj2MoWg1OznPuFOdP8YELfPQxTs5tOfD+ApqTs3zuw1w8w6/+gP4N/Zh+TJrBVIE8CEULnqLV5OJZ/vxJPnKJ9y5Sr9x/5P8DMo5/sshpyMoAAAAASUVORK5CYII="
```
Comment (jsnajdr) [inline packages/odie-client/src/assets/human-avatar.tsx:24]: This image should be extracted to a `.png` file. What if I want to see how the PNG image looks like?

Webpack has capabilities (with `file-loader` and `url-loader`) to import such a file and convert it into an inline data URL. The output of the build can be identical to the current state, but the source can be managed in a much nicer way where JavaScript is in `.js` files and images are in `.png` files.

Comment (arcangelini) [reply packages/odie-client/src/assets/human-avatar.tsx:24]: Thanks for pointing this out @jsnajdr I honestly didn't even inspect the SVG much. This was copied directly from the Figma. We have a follow-up PR to work on the stylings and will take this into account and address this.

**Comment by a8ci18n** [comment]
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/16912202).

**Pull request merged**